e2e/cypress/integration/channel/message_parse_spec.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Not mentioned in the ticket but please move this file to `e2e/cypress/integration/messaging`

Reply:
will do